onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -232,6 +249,29 @@ def aten_addmm(
     return op.Add(scaled_self, scaled_mat1_mat2)
 
 
+@torch_op("aten::addmm")
+def aten_addmm_gemm(
+    self: FLOAT, mat1: FLOAT, mat2: FLOAT, beta: float = 1.0, alpha: float = 1.0
```

Comment:
Why only do 'gemm' variant on float inputs?

Reply:
ORT implements Gemm for float inputs only. So I set it accordingly for practicality

Reply:
Changed to TFloat because it supports float16 as well